pokemonterminal/wallpaper/adapters/__init__.py

```diff
@@ -1,17 +1,21 @@
-class WallpaperProvider:
+from abc import ABC, abstractmethod
+
+class WallpaperProvider(ABC):
     """
     Interface representing all the different desktop environments supported
     by pokemon-terminal if you want to implement a DE, create a module in this
     folder that implements this interface, reflection will do the rest.
     """
 
+    @abstractmethod
     def change_wallpaper(path: str):
```

Comment:
I can't believe I haven't noticed this sooner, but since you're changing this, could you please either: make these methods explicitly static with `@staticmethod`, or add self?